jdk/src/share/classes/java/util/concurrent/ConcurrentHashMap.java

```diff
@@ -3466,748 +3465,6 @@ public int characteristics() {
         }
     }
 
-    // Parallel bulk operations
-
-    /**
-     * Computes initial batch value for bulk tasks. The returned value
-     * is approximately exp2 of the number of times (minus one) to
-     * split task by two before executing leaf action. This value is
-     * faster to compute and more convenient to use as a guide to
-     * splitting than is the depth, since it is used while dividing by
-     * two anyway.
-     */
-    final int batchFor(long b) {
-        long n;
-        if (b == Long.MAX_VALUE || (n = sumCount()) <= 1L || n < b)
-            return 0;
-        int sp = ForkJoinPool.getCommonPoolParallelism() << 2; // slack of 4
```

Comment:
FJP seems to only be used on this line. Perhaps just use some number like 16, then we don't need to drop the whole class?

Reply:
Since the DJVM is going to be single-threaded, I think the correct number to use here would be 1. But I'm only dropping the `BulkTask` functions from `ConcurrentHashMap` anway.